spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/AbstractLocalDeployerSupport.java

```diff
@@ -42,6 +40,8 @@
  */
 public abstract class AbstractLocalDeployerSupport {
 
+	private static final Logger logger = LoggerFactory.getLogger(LocalAppDeployer.class);
```

Comment:
Instead of setting `LocalAppDeployer` class name here, can we have something like this:
```
protected final Logger logger = LoggerFactory.getLogger(getClass());
```